docs/units/quantity.rst

```diff
@@ -171,10 +171,10 @@ Plotting a quantity with an incompatible unit will raise an exception:
 
 .. doctest-requires:: matplotlib
 
-    >>> plt.plot([1, 2, 3] * u.kg)  # doctest: +IGNORE_EXCEPTION_DETAIL
+    >>> plt.plot([1, 2, 3] * u.kg)  # doctest: +SKIP
```

Comment:
The original doctest already ignore pretty much everything, so this change officially skips the check. 😬 

Unless `doctest-plus` is smart enough to allow me to ignore the exception type and only check for error message, @drdavella ?

Reply:
My sense would be to remove the whole example and just state that incompatible units "will raise an exception."

Reply:
@mhvk , please see #8361 .